Helper/Data.php

```diff
@@ -704,8 +718,13 @@ public function getWsUsername($storeId = null)
      */
     public function getLiveEndpointPrefix($storeId = null)
     {
-        $prefix = trim($this->getAdyenAbstractConfigData('live_endpoint_url_prefix', $storeId));
-        return $prefix;
+        $prefix = $this->getAdyenAbstractConfigData('live_endpoint_url_prefix', $storeId);
+
+        if (is_null($prefix)) {
+            return false;
```

Comment:
Let's turn `string` configurations into `string|null` to represent lack of setup:
```suggestion
            return null;
```